.circleci/config.yml

```diff
@@ -258,11 +259,11 @@ commands:
           name: Install CI dependencies
           command: |
             cd ..
-            command -v sccache >/dev/null || rustup run --install stable cargo install sccache
-            command -v cargo-cache >/dev/null || rustup run --install stable cargo install cargo-cache
-            command -v cargo2junit >/dev/null || rustup run --install stable cargo install cargo2junit
+            command -v sccache >/dev/null || cargo install sccache
```

Comment:
Is there any advantage to using sccache on CLI?
Not suggesting any change as part of this PR just a question.

Reply:
It's actually all invoked via the CLI; this is just a direct invocation. sccache is integrated into the Cargo build by setting `RUSTC_WRAPPER=sccache`, which this config exports in line 93.